services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/toggles/FeatureTogglesV2Spec.groovy

```diff
@@ -282,4 +285,71 @@ feature toogle"() {
         }
         flowHelperV2.deleteFlow(flow.flowId)
     }
+
+    @Tidy
+    @Tags(LOW_PRIORITY)
+    def "System doesn't reroute flow when flows_reroute_on_isl_discovery: false"() {
+        given: "A flow with alternative paths"
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
+            it.paths.size() >= 2
+        } ?: assumeTrue("No suiting switches found", false)
+        def allFlowPaths = switchPair.paths
+        def flow = flowHelperV2.randomFlow(switchPair)
+        flowHelperV2.addFlow(flow)
+
+        //you have to break all altPaths to avoid rerouting when flowPath is broken
+        def flowPath = pathHelper.convert(northbound.getFlowPath(flow.flowId))
+        def altPaths = allFlowPaths.findAll { it != flowPath && it.first().portNo != flowPath.first().portNo }
+        List<PathNode> broughtDownPorts = []
+        altPaths.unique { it.first() }.each { path ->
+            def src = path.first()
+            broughtDownPorts.add(src)
+            antiflap.portDown(src.switchId, src.portNo)
+        }
+        def altPortsAreDown = true
+
+        and: "Set flowsRerouteOnIslDiscovery=false"
+        northbound.toggleFeature(FeatureTogglesDto.builder()
+                .flowsRerouteOnIslDiscoveryEnabled(false)
+                .build())
+        def featureToogleIsUpdated = true
+
+        when: "Break the flow path(bring port down on the src switch)"
+        def getInvolvedIsls = pathHelper.getInvolvedIsls(flowPath)
+        def islToBreak = getInvolvedIsls.first()
+        antiflap.portDown(islToBreak.srcSwitch.dpId, islToBreak.srcPort)
+        def mainPortIsDown = true
+
+        then: "The flow becomes 'Down'"
+        Wrappers.wait(discoveryTimeout + rerouteDelay + WAIT_OFFSET * 2) {
+            assert northbound.getFlowStatus(flow.flowId).status == FlowState.DOWN
+        }
+
+        and: "Flow is not rerouted"
```

Comment:
This is misleading. Flow reroute is initiated, but the reroute fails since there are no paths left

Reply:
agree, deleted